tests/ecosystem/upgrade/test_upgrade.py

```diff
@@ -88,3 +96,22 @@ def test_check_mon_pdb_post_upgrade():
     # The PDB values are considered from OCS 4.5 onwards.
     assert disruptions_allowed == 1, "Mon Disruptions_allowed count not matching"
     assert min_available_mon == 2, "Minimum available mon count is not matching"
+
+
+@post_ocp_upgrade
+@post_ocs_upgrade
+@pytest.mark.polarion_id("OCS-2524")
+def test_check_osd_pdb_post_upgrade():
```

Comment:
Please move this under "tests/manage/z_cluster/upgrade/"

Reply:
Moved both tests under same file